modules/apps/27-interchain-accounts/keeper/relay_test.go

```diff
@@ -0,0 +1,83 @@
+package keeper_test
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
+	ibctesting "github.com/cosmos/ibc-go/testing"
+)
+
```

Comment:
nit: separate imports like such
```go
import (
    // standard library imports
    "fmt"
    
    // external library imports
    "github.com/tendermint/tendermint"
    "github.com/cosmos/cosmos-sdk"
    
    // ibc-go imports
    "github.com/cosmos/ibc-go"
)
```

This is actually completely inconsistent throughout the repository because of the migration from the sdk -> ibc-go, but I think it is nice having a simple standard like this and fix it as we go forward

Reply:
updated :+1: